source/educators/how-tos/add_delete_course_tags

```diff
@@ -0,0 +1,46 @@
+Add and delete tags on courses
```

Comment:
@crathbun428 this file needs to be deleted - I think you need to delete it _and then_ commit the change.